paparazzi/src/main/java/app/cash/paparazzi/internal/compose/RecomposerWatcher.kt

```diff
@@ -0,0 +1,87 @@
+@file:OptIn(ExperimentalComposeUiApi::class)
+
+package app.cash.paparazzi.internal.compose
+
+import androidx.compose.runtime.Recomposer
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.InternalComposeUiApi
+import androidx.compose.ui.platform.WindowRecomposerPolicy
+import androidx.compose.ui.platform.createLifecycleAwareWindowRecomposer
+import app.cash.paparazzi.PaparazziSdk.Companion.isPresentInClasspath
+import app.cash.paparazzi.internal.PaparazziLogger
+import com.android.ide.common.rendering.api.Result
+import com.android.ide.common.rendering.api.Result.Status.ERROR_UNKNOWN
+import kotlin.coroutines.CoroutineContext
+
+@InternalComposeUiApi
+internal class RecomposerWatcher(
```

Comment:
given the limited use case, extracting this to its own class is a bit overkill, imo.  I really liked where https://github.com/cashapp/paparazzi/commit/505720f93df6685f44d40870ee0637065c0614f2 was heading, let's stick with that!